exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetScanBatchCreator.java

```diff
@@ -87,9 +87,6 @@ public ScanBatch getBatch(FragmentContext context, ParquetRowGroupScan rowGroupS
           newColumns.add(column);
         }
       }
-      if (newColumns.isEmpty()) {
```

Comment:
@amansinha100 , I made slightly change to the patch to address the comments. Could you please take another look?

Thanks!